src/views/mentorship/mentorship-hero/ui/mentorship-hero.module.scss

```diff
@@ -1,76 +1,27 @@
-.hero-page {
-  gap: 16px;
+.hero {
   min-height: 440px;
-  background: $color-yellow;
+  background: url('../../../../shared/assets/mentorship-hero.webp') center / cover no-repeat;
 
-  &-content {
+  &-info {
     display: flex;
-    flex-direction: row;
-    align-items: center;
-    justify-content: space-between;
-
-    .title-container {
-      display: flex;
-      flex-direction: column;
-      width: fit-content;
-      text-align: left;
-
-      .title-main {
-        @include media-tablet {
-          font-size: 80px;
-        }
-
-        @include media-mobile-landscape {
-          font-size: 40px;
-        }
-      }
-
-      .subtitle-container {
-        display: flex;
-        flex-direction: row;
-        justify-content: space-between;
+    flex-direction: column;
 
-        @include media-tablet {
-          flex-direction: column;
-        }
-      }
+    width: fit-content;
+    padding: 32px;
 
-      .description-title {
-        @include media-tablet {
-          font-size: 24px;
-          line-height: 32px;
-        }
-      }
-    }
+    text-align: left;
 
-    .sloth-mascot {
-      max-width: 300px; // TODO change if sloth pic will be changed
-      height: auto;
-      max-height: 235px;
-      object-fit: contain;
+    background-color: rgba($color-white, $opacity-80);
```

Comment:
Please, use `hsla` format because we use it everywhere on the project
![image](https://github.com/user-attachments/assets/3667b1f9-c9fa-444c-b1d3-8244cea6ba96)

Reply:
Done